pkgs/development/ruby-modules/gem/default.nix

```
@@ -218,7 +218,9 @@ stdenv.mkDerivation ((builtins.removeAttrs attrs ["source"]) // {
     # looks like useless files which break build repeatability and consume space
     pushd $out/${ruby.gemPath}
     rm -fv doc/*/*/created.rid || true
-    rm -fv {gems/*/ext/*,extensions/*/*/*}/{Makefile,mkmf.log,gem_make.out} || true
+    shopt -s globstar
+    rm -fv {gems/*/ext/**/,extensions/**/}/{Makefile,mkmf.log,gem_make.out} || true
```

```suggestion
    find gems/*/ext/ extensions/ -name Makefile -o -name mkmf.log -o -name gem_make.out -delete
```
Something like that would also handle cases where there are more than (1024 / 3) extensions.

---

included the `created.rid` from above as well

btw: I didn't know that `find -iname hello -o -iname world -delete` only deletes for the final boolean!

---

thanks, I learned something as well